src/code/z_actor.c

```diff
@@ -391,12 +391,12 @@ void func_800B4B50(Actor* actor, Lights* mapper, PlayState* play) {
     }
 }
 
-void Actor_GetProjectedPos(PlayState* play, Vec3f* arg1, Vec3f* arg2, f32* arg3) {
-    SkinMatrix_Vec3fMtxFMultXYZW(&play->viewProjectionMtxF, arg1, arg2, arg3);
-    if (*arg3 < 1.0f) {
-        *arg3 = 1.0f;
+void Actor_GetProjectedPos(PlayState* play, Vec3f* worldPos, Vec3f* projectedPos, f32* invW) {
+    SkinMatrix_Vec3fMtxFMultXYZW(&play->viewProjectionMtxF, worldPos, projectedPos, invW);
+    if (*invW < 1.0f) {
+        *invW = 1.0f;
     } else {
-        *arg3 = 1.0f / *arg3;
+        *invW = 1.0f / *invW;
```

Comment:
What's the motivation for `invW` instead of `cappedInvW`? Note that many of these changes here directly clash with https://github.com/zeldaret/mm/pull/933 so we should try to sync these

Reply:
I don't think it being capped is a crucial part of the operation, rather than just avoiding dividing by something small.